docs/migrations.md

```diff
@@ -0,0 +1,69 @@
+# v1alpha3 HAProxy to v1alpha3 kube-vip
+
+to migrate from an existing `HAProxyLoadBalancer` to `kube-vip` you will need the following steps:
```

Comment:
What about adding a little blurb explaining _why_ the user should do this operation
- HA proxy being deprecated and it will be removed in v1alpha4 (at least AFAIK)
- With kube-vip you "save" 1 machine